src/Form/inputTypeConfig.js

```diff
@@ -23,14 +23,15 @@ const getIdentitiesIcon = (input, {device}) => {
 
 /**
  * A map of config objects. These help by centralising here some complexity
- * @type {Object<SupportedMainTypes, InputTypeConfig>}
+ * TODO: Why is `emailNew` not part of 'SupportedMainTypes'?
```

Comment:
We're removing `emailNew` everywhere. The new thing is `identities.emailAddress`. We still have to backport this properly to other platforms.

Reply:
ok, changing the comment to reflect this